pages/community/precice-workshop-organization.md

```diff
@@ -0,0 +1,116 @@
+
+Multiple types of tickets are typically defined:
+
+- early -vs- late registration
+- full price -vs- discount for academia
+- training course included -vs- only main part of the workshop
+
+In addition, free registration is accounted for:
+
+- holders of a support license
+- invited speakers
+- special cases
+
+### Set milestones
+
+Set clear responsible persons for each task, and track progress in a collaborative project board.
+
+Important dates to set:
+
+- "save the date" announcement: one year ahead
+- call for contributions (needs a form): open six months ahead. An early call mainly acts as advertisement.
+- open registration (needs a form): six months ahead. Early registration is important for participants that need a visa application to travel.
+- abstract submission deadline and call acceptance: the earlier the better. Can be after the early-bird registration deadline. The call acceptance can be as short as one week after the deadline.
+- early-bird registration deadline: important for estimating the number of participants, driven by the catering requirements.
+- request catering offers: typically 1-2 months before the workshop.
+- reserve tables for the workshop dinner: typically 1-2 months before the workshop.
+- late registration deadline: typically one week before the workshop, does not affect much.
+
+## Registration infrastructure
+
+Two forms are mainly needed, and it is easier if they are decoupled: an abstract submission form, and a registration form.
+
+The abstract submission form should collect:
+
+- Kind of contribution (talk or poster, if offered)
+- Presenter
+- Affiliation
+- Contact email
+- Title
+- Abstract (plain text, short)
+- A disclaimer that, if accepted, this information is going to appear on the website
+
+This form can be implemented in any survey tool. Live access to the data by the preCICE team would be helpful, to understand where further advertising might be needed.
+
+The registration form should collect:
+
+- Name
+- Affiliation
+- Contact email
+- Dates/modules that one plans to attend:
+  - Training course
+  - Main workshop
+  - Dinner
+  - User support session (if the only event of the day)
+  - Potential other social events
+- Permission to include contact details in a participants list distributed to the participants
+- A disclaimer that photos are going to be taken on site and will be used for advertising
+- Potential questions of interest for the workshop (particular topics of interest, goals)
+
+This form is typically implemented by the entity processing the payments, which also issues invoices, receipts. Who issues confirmation of participation depends on the local infrastructure available. If not automatic, the organizers can provide a letter at the end to those requesting it.
```

Comment:
```suggestion
This form is typically implemented by the local organizers. Payments including invoices or receipts could be part of the process or separate.
```
So far, INFOS was the only entity that directly included payments.

With confirmation, you mean letter of attendance here, right? Could be confusing like this.